src/Utils/Validate.js

```diff
@@ -4,8 +4,11 @@ const Required = (value) => {
   return 'Required!';
 };
 
-const RequiredInput = () => {
-  return false;
+const isURLValid = (value) => {
+  const regex = new RegExp('^$|([Hh][Tt][Tt][Pp]|[Ff][Tt][Pp])([Ss])?://.+$');
```

Comment:
I think this const could be on top level, out of the function. And maybe it could have a self-explaining name, like `REGEXP_URL` or better